lms/djangoapps/verify_student/services.py

```diff
@@ -117,11 +123,14 @@ def get_expiration_datetime(cls, user, statuses):
             'status__in': statuses,
         }
 
+        id_verifications = VerificationAttempt.objects.filter(**filter_kwargs)
```

Comment:
Should there be something in front of id_verifications here to specify what kind of id verification this is? Is persona_id_verifications too specific here?

Reply:
IMO this is intended to be the most general verification model so it doesn't need further specification